configure-nonroot.sh

```diff
@@ -23,7 +23,7 @@ chown "${IRONIC_USER}":"${IRONIC_GROUP}" /shared
 # that need to have correct ownership as the entire ironic in BMO
 # deployment shares a single fsGroup in manifest's securityContext
 mkdir -p /certs/ca
-chown "${IRONIC_USER}":"${IRONIC_GROUP}" /certs{,/ca}
+chown -R "${IRONIC_USER}":"${IRONIC_GROUP}" /certs{,/ca}
```

Comment:
On an another look, the original syntax does expand to cover both directories anyways, so it should be possible to create `/certs/ca/ironic`. Have you tested this actually fixes your issue?

Reply:
I am trying to build an OpenShift ironic-image to test this locally, will mark the PR as a WIP